changelogs/fragments/33-vmware_guest_info_list_fix.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
it's more a `minor_changes` IMO.

Reply:
It's a bugfix since user specified property is a list and module API is not handling it properly.

Reply:
thanks for the clarifiation.